src/transaction/log_manager.c

```diff
@@ -321,6 +321,9 @@ static cubthread::daemon *log_Check_ha_delay_info_daemon = NULL;
 
 static cubthread::daemon *log_Flush_daemon = NULL;
 static std::atomic_bool log_Flush_has_been_requested = {false};
+
+static cubthread::daemon *cdc_log_Producer_daemon = NULL;
+static cubthread::daemon *cdc_Thread_checker_daemon = NULL;
```

Comment:
Could you tell me the purpose of this daemon? I wasn't told it in a design review?